mps_youtube/player.py

```diff
@@ -134,7 +135,7 @@ def _mplayer_help(short=True):
     return lines.format(c.g, c.w)
 
 
-def _playsong(song, failcount=0, override=False, softrepeat=False):
+def stream_details(song, failcount=0, override=False, softrepeat=False):
     """ Play song using config.PLAYER called with args config.PLAYERARGS."""
     # pylint: disable=R0911,R0912
     if not config.PLAYER.get or not util.has_exefile(config.PLAYER.get):
```

Comment:
It should only test for a player when actually playing. Nor should the `NOTIFIER` be called.